htdocs/core/class/html.form.class.php

```diff
@@ -1055,8 +1055,12 @@ public function select_incoterms($selected = '', $location_incoterms = '', $page
 				}
 			}
 			$out .= '</select>';
-
-			$out .= '<input id="location_incoterms" class="maxwidth100onsmartphone nomargintop nomarginbottom" name="location_incoterms" value="'.$location_incoterms.'">';
+			
```

Comment:
You should configure your ide to auto remove whitespace at end of line and to use tabs instead of spaces for tabs.